### ISSUE TITLE
main function identification does not always jump to main function

### ISSUE THREAD

**Issue by op2786** [Vector35/binaryninja-api#4974]
**Version and Platform (required):**
- Binary Ninja Version: 3.6.4799-dev, 8197e491
- OS: windows
- OS Version: 10
- CPU Architecture: x86_64

**Bug Description:**
I noticed that main function identification does not always jump to the main function. I can see from the logs that it found the main function, but does not navigates to it. 

**Steps To Reproduce:**
1. Load a x64 windows binary (I did not test other platforms)
2. Make sure it finds the main function from the logs
3. If it navigates to main function, reopen same binary until you trigger the issue

If this is not enough for reproducing the issue, I can share my binary.

**Expected Behavior:**
I expect BN to always navigate to the main function if it finds it.

**Screenshots:**
In the first log entry, it navigated to the main function that it found but in second entry it stayed on _start function.
![image](https://github.com/Vector35/binaryninja-api/assets/109607896/6837382a-ed97-4744-a27f-74e393cf73cc)

**Comment by plafosse** [comment]
Have you ever opened this particular binary before? If so your previous location has been saved will be restored.

**Comment by op2786** [comment]
Good point but I've disabled restore view state option before doing this test. So this should be not related to it.

**Comment by xusheng6** [comment]
@op2786 I cannot reproduce this, please share the binary with me in private. thx!

**Comment by xusheng6** [comment]
Vector35 folks should search `orchestra sherifian inclip queme conjugates` to find the file

**Comment by xusheng6** [comment]
@op2786 I am still unable to reproduce this, can you record a video of it?

**Comment by op2786** [comment]
I am unable to reproduce it on my mac. Could it possibly be related to the fact that I'm running windows on a virtual machine? Perhaps it's somehow connected to the sluggish performance of Windows on the VM?

**Comment by plafosse** [comment]
As we still don't have solid repro steps this one is being marked as Unreproducible.

**Comment by fuzyll** [comment]
@xusheng6 if you've tried this on Windows and can't reproduce it, let's just close it out. It can be reopened later if we have more solid repro steps.

If you *can* repro it, let's get the ticket's tags updated. 🙂

**Comment by xusheng6** [comment]
@fuzyll @plafosse I have a repro. It only reproduces on Windows. I have updated the tags
